### PR TITLE
fix(balance report): Fix the reduce value computed in the balance report

### LINKED ISSUE #1693: (bug) general ledger and balance report are corrupted
[ ] General ledger :
	- Bug: The amount of the balance doesn't correspond to the sold of the account
	- Fix: the french translation "Grande Livre" to "Grand Livre"

[ ] Report / Balance : 
	- Bug: The report is crashed

[ ]  Report / Balance agee des creances :
	- Bug: Error in the console

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR fix the issue #1693 

Closes #1693

### CODE REVIEW

File: client/src/modules/reports/modals/agedCreditors.modal.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
       <input class="form-control" name="label" autocomplete="off" ng-model="ReportConfigCtrl.label" required />
 
       <div class="help-block" ng-messages="ConfigForm.label.$error" ng-show="ConfigForm.$submitted">
-        <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
```
Comment (jniles) [inline client/src/modules/reports/modals/agedCreditors.modal.html:16]: :+1:

File: server/controllers/finance/reports/balance/index.js [hunk 2]
```diff
@@ -124,7 +124,7 @@ function processAccounts(balances, a, totals) {
       t.middleCredit += (account.middleCredit || 0);
       t.endDebit += (account.endDebit || 0);
       t.endCredit += (account.endCredit || 0);
-      return totals;
+      return t;
```
Comment (jniles) [inline server/controllers/finance/reports/balance/index.js:127]: :dancer:

**Review by jniles — APPROVED** [review]
Looks good to me :+1:

We should migrate these types of sums to SQL in the future.  See https://github.com/IMA-WorldHealth/bhima-2.X/issues/1145 for an issue on a report concerning this.

**Comment by jniles** [comment]
Also :+1: this is a much better commit message structure.   It probably would be better to simply say:

> fix(balance): fix typo in reduce value
> 
> On the server side, the reduce variable was renamed and caused the server to crash.  It is renamed to the appropriate name.
>
> Closes #1693.

the "server side" detail can go into body of the commit message instead of header.  Also, you must write "Closes #XXX" or "Fixes #YYY" for github to actually close those.

**Comment by jniles** [comment]
@mbayopanda, can you rebase this to master?

**Comment by mbayopanda** [comment]
@jniles @sfount could you review this PR please...

File: client/src/modules/reports/modals/agedCreditors.modal.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="modal-header">
   <ol class="headercrumb">
-    <li class="title" translate> ReportConfigCtrl.report.title_key </li>
+    <li class="title" translate> {{ ReportConfigCtrl.report.title_key }} </li>
```
Comment (jniles) [inline client/src/modules/reports/modals/agedCreditors.modal.html:3]: :dancer:

**Comment by jniles** [comment]
@sfount, I'll leave this to you to merge.  I recommend using bors :smiley:

**Comment by sfount** [comment]
:+1: LGTM @mbayopanda @jniles

**Pull request merged**